data-loader/core/src/main/java/com/scalar/db/dataloader/core/util/ConfigUtil.java

```diff
@@ -0,0 +1,98 @@
+package com.scalar.db.dataloader.core.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Utility class for loading and retrieving configuration properties.
+ *
+ * <p>This class reads properties from a {@code config.properties} file located in the classpath.
+ */
+public class ConfigUtil {
+  public static final String CONFIG_PROPERTIES = "config.properties";
```

Comment:
So, are we using a fixed configuration file? Could you please explain why we don’t use command-line arguments for the queue size and thread pool size?

Anyway, if we’re using a file for the configurations, I think we should pass the configuration file name via command-line arguments.

Reply:
> I think we should pass the configuration file name via command-line arguments.

👍

Reply:
@brfrn169 san, @komamitsu san,
Thank you for the suggestions.
I have updated to use command line arguments to configure queue size(added new parameter) and thread pool size (there was an existing parameter for this, but I didn't use that as the initial new data loader changes had usage of number of cores in it. So I didn't change that). I have removed configurable by properties file completely. Asking the users to add a properties file for just 2 parameter while rest are passed as arguments seemed confusing. I have also remove `numCores` from method arguments and instead use the value directly as instructed.